src/annotation.ts

```diff
@@ -69,6 +69,12 @@ export class Annotation extends Modifier {
     centerStem: AnnotationVerticalJustify.CENTER_STEM,
   };
 
+  // Use the same padding for annotations as note head so the
+  // words don't run into each other.
+  static get minAnnotationPadding(): number {
+    const musicFont = Tables.currentMusicFont();
+    return musicFont.lookupMetric('glyphs.noteHead.minPadding');
+  }
```

Comment:
`StaveNote::minNoteheadPadding()`makes the same calculation. You could use it. 
Is this calculation good for `TabNotes`?

Reply:
@AaronDavidNewman looks like your typescript (that's building the ESM) doesn't know the `??` operator, I've seen it happen in other systems, maybe an old version? A workaround is rewriting `??` to ` x ? y : z`, but maybe you can use a newer typescript version.

Reply:
> > StaveNote.minNoteheadPadding()makes the same calculation. You could use it.
> > Is this calculation good for TabNotes?
> 
> I didn't want to add a dependency to StaveNote from Annotation. I think it will look good for any type of notes, it's to gap vs. other text mostly (lyrics) so words aren't squished together.

I see, what about calling `Tables.currentMusicFont().lookupMetric('glyphs.noteHead.minPadding')` directly on the two instances below?
 
> > try deleting node_modules rm -r node_modules/ and installing again npm install
> 
> This happened even on a fresh install. I was using `npm run reference`.

what version of npm do you use? `npm --version` Do you have typescript installed globally? `tsc --version`

Reply:
npm version: 6.14.6.  I see the version referenced in the package is 8.3.  node version is 12.18.3.

When I installed this, the latest node (16.something) didn't work.  It had something to do with a dependency on the 'canvas' package.  Maybe this is not an issue anymore.  I'll try updating node also.

Reply:
try `sudo npm install -g npm@latest`

Reply:
OK, finally got it working.   Running npm install with latest node and npm did the trick.
It is pretty much all like this.  The 120+ notes with annotations take up slightly more room.  So no surprises:

was:
![Grace_Notes With_Articulation_and_Annotation_on_Parent_Note Bravura_reference](https://user-images.githubusercontent.com/5438280/158704698-6ade7ee6-31b6-4494-bf54-6665d064d038.png)

is now (grace note before words is a little further to the right):
![Grace_Notes With_Articulation_and_Annotation_on_Parent_Note Bravura_current](https://user-images.githubusercontent.com/5438280/158704707-7b7c99d8-2ee3-4500-bc42-e57af59228f5.png)

![pptr-Annotation Standard_Notation_Annotation Petaluma svg](https://user-images.githubusercontent.com/5438280/158704786-38bb2d6f-1bcf-4b44-ae8d-bd97f6f2b9c5.png)


![pptr-canvas_Annotation Harmonics Bravura](https://user-images.githubusercontent.com/5438280/158704808-8757a224-a8c3-43eb-8fea-e36d8a8b61bd.png)